modules/site-health/admin-site-health.php

```diff
@@ -246,6 +246,17 @@ public function info_options( $debug_info ) {
 			}
 		}
 
+		//Multisite
+		if ( is_multisite() ){
+			$plugin = POLYLANG_ROOT_FILE;
+			$network_activated = __( 'No', 'polylang' );
+			if ( is_plugin_active_for_network( 'polylang/polylang.php' ) || is_plugin_active_for_network( 'polylang-pro/polylang.php' )){
+				$network_activated = __( 'Yes', 'polylang' );
```

Comment:
I know translations are more performant now however an `if() ... else` would avoid you to translate 2 stings when only one is necessary.